backend/typescript/rest/adminRoutes.ts

```diff
@@ -0,0 +1,35 @@
+import { Router } from "express";
+
+import AdminService from "../services/implementations/adminService";
+import IAdminService from "../services/interfaces/adminService";
+import { getErrorMessage } from "../utilities/errorUtils";
+import { sendResponseByMimeType } from "../utilities/responseUtil";
+import { WaiverDTO } from "../types";
+
+const adminRouter: Router = Router();
+const adminService: IAdminService = new AdminService(); 
+
+
+
+adminRouter.post("/waiver", async (req, res) => {
+    try {
+      console.log(req.body)
+      const waiver = await adminService.updateWaiver({
+          clauses: req.body
+      })
+      res.status(200).json(waiver);
```

Comment:
this can be changed to `waiver.clauses` if we only want to return the array of clauses